health/egov-indexer/attendance-service-indexer.yml

```
@@ -0,0 +1,93 @@
+          jsonPath: $.*
+          timeStampField: $.attendanceLog.auditDetails.lastModifiedTime
+
+    - topic: update-attendance-log-health
+      configKey: INDEX
+      indexes:
+        - name: attendance-log-index-v1
+          type: attendance-log
+          id: $.id
+          isBulk: true
+          jsonPath: $.*
+          timeStampField: $.time
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)